components/Form.js

```diff
@@ -46,6 +48,64 @@ const RangeInputLabels = styled.div`
   justify-content: space-between;
 `;
 
+const VisuallyHidden = styled.input`
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
```

Comment:
what does this exactly do ?

Reply:
Thanks, you are right, this is not needed. We will take care of this in one of the following tickets.